CHANGELOG.md

```diff
@@ -51,21 +58,24 @@ Added validate_grant_permission() and validate_revoke_permission() methods per S
 This allows users to download SNS canister WASMs.
 
 ### fix: fixed error text
+
 - `dfx nns install` had the wrong instructions for setting up the local replica type
 
 ### fix: creating an identity with `--force` no longer switches to the newly created identity
 
 ### feat(frontend-canister)!: reworked to use permissions-based access control
 
 The permissions are as follows:
-- ManagePermissions: Can grant and revoke permissions to any principal.  Controllers implicitly have this permission.
+
+- ManagePermissions: Can grant and revoke permissions to any principal. Controllers implicitly have this permission.
```

Comment:
Would you please submit these unrelated markdown-correctness changes as a separate PR?   It's good to clean them up, but they would be better put in a dedicated PR.